### PR TITLE
Added .features + example

### PR DESCRIPTION
**Pull request by ColinMaudry**
It took some time until I figured out the name of the field `features` came from the matching BT in eForms!

This PR adds:

- `features` in schema
- legal context for `features` and `acceleratedProcedure`
- example for `features`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,19 @@
 
 This extension adds a block to decribe the procurement procedure.
 
+## Legal context
+
+In the European Union, this extension's fields correspond to [eForms BT-88, BT-106, BT-1351](https://github.com/eForms/eForms) and [Article 93 and 45 clause 3 of Directive 2014/25/EU](https://eur-lex.europa.eu/eli/dir/2014/25/oj). See [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/master/en/) for the correspondences to Tenders Electronic Daily (TED).
```
Comment (jpmckinney) [inline README.md]: -> Articles 95 and 45(3)

File: release-schema.json [hunk 2]
```diff
@@ -29,6 +29,14 @@
             "string",
             "null"
           ]
+        },
+        "features": {
+          "title": "Features",
+          "description": "The national rules that apply to the procedure.",
```
Comment (jpmckinney) [inline release-schema.json]: Let's use the longer description from eForms:

> The main features of the procedure (e.g. description of the individual stages) and information about where the full rules for the procedure can be found.

File: README.md (hunk 1 above)
Comment (ColinMaudry) [reply README.md]: Done

File: release-schema.json (hunk 2 above)
Comment (ColinMaudry) [reply release-schema.json]: Done

**Pull request merged**